assignments/objects.js

```diff
@@ -18,27 +18,68 @@ const example = {
 }
 
 // Write your intern objects here:
-
+const mitzi = { 
+  id: 1, 
+  first-name: "Mitzi", 
+  email: "mmelloy0@psu.edu",
+  gender: "F", 
+};
+
+const kennan = {
+  id: 2,
+  first-name: "Kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M",
+  speak: function(){
+    return "Hello, my name is Kennan!";
+  }
+};
+
+const keven = {
+  id: 3,
+  first-name: "Keven",
+  email: "kmummery2@wikimedia.org",
+  gender: "M",
+};
+
+const gannie = {
+  id: 4,
+  first-name: "Gannie",
+  email: "gmartinson3@illinois.edu",
+  gender: "M",
+};
+
+const antonietta = {
+  id: 5,
+  first-name: "Antonietta",
+  email: "adaine5@samsung.com",
+  gender: "F",
+  multiplyNums: function(num1, num2){
+    return num1 * num2;
+  }
+};
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(mitzi.name);
```

Comment:
name doesn't exist as one of the keys inside your object. In your object, I see a `first-name`. The probem with using the special character `-`, is you have to use the bracket notation for you to refer to it.